### PR TITLE
PLT-4020 Message is displayed when user joins Off_Topic

### PR DESCRIPTION
**Pull request by R-Wang97**
#### Summary
A system message is now shown when a user joins the Off-Topic channel. 

#### Ticket Link
https://mattermost.atlassian.net/browse/PLT-4020

#### Checklist
- [x] Added or updated unit tests (required for all new features)
- [x] Has UI changes

### CODE REVIEW

File: config/config.json [hunk 1]
```diff
@@ -52,7 +52,7 @@
         "MaxIdleConns": 10,
         "MaxOpenConns": 10,
         "Trace": false,
-        "AtRestEncryptKey": ""
+        "AtRestEncryptKey": "ixrdftmiyw5k4zjz1kprd73iaunow5c6"
```
Comment (enahum) [inline config/config.json]: Just a quick note @R-Wang97,

can you remove this value in the config file?

File: config/config.json [hunk 2]
```diff
@@ -76,7 +76,7 @@
         "DriverName": "local",
         "Directory": "./data/",
         "EnablePublicLink": false,
-        "PublicLinkSalt": "",
+        "PublicLinkSalt": "i3bkecwyggquxzntdowz6hpqnsqzmhqr",
```
Comment (enahum) [inline config/config.json]: also this one

File: config/config.json [hunk 3]
```diff
@@ -107,8 +107,8 @@
         "SMTPServer": "",
         "SMTPPort": "",
         "ConnectionSecurity": "",
-        "InviteSalt": "",
-        "PasswordResetSalt": "",
+        "InviteSalt": "3jcxrw1uhfnzampwagg4m5f8d9aq1mao",
+        "PasswordResetSalt": "3hxtzgewdi3b84txezrr9cfgbympazgh",
```
Comment (enahum) [inline config/config.json]: and these two

**Comment by R-Wang97** [comment]
@enahum Removed those entries from the config file, sorry about that.

**Comment by jasonblais** [comment]
Thanks @R-Wang97! 

+1, see screenshot of test cases on Windows 10, Chrome.

Also tested other channels and private groups, all seems to work

image

PS: Found a bug unrelated to the PR, filed here: https://mattermost.atlassian.net/browse/PLT-4281

**Comment by DavidLu1997** [comment]
+1

**Pull request merged**